linalg/densemat.cpp

```diff
@@ -3971,7 +4047,12 @@ void LUFactors::Factor(int m)
             }
          }
       }
-      MFEM_ASSERT(data[i+i*m] != 0.0, "division by zero");
+
+      if ( abs(data[i+i*m]-0.0) <= TOL )
```

Comment:
In most of MFEM we don't put space after `(` and before `)`. I know it's nitpicking but it's good to keep the overall style of the project.

Reply:
@tzanio do you want us to point out these style inconsistencies? I didn't mention those on purpose to keep the contributing bar low and astyle did not complain. On the other hand my own opinion is leaning towards @jakubcerveny.

Reply:
I think it is worth pointing these out for new contributors so they get introduced to our typical conventions. Sometimes I make the changes myself (and I assume they see the diffs). Unfortunately astyle is limited in what it could enforce...

Reply:
Done.